META.d/_summary.yml

```
@@ -0,0 +1,11 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+schema: 1.1
+partition: tf-ecosystem
+category: library
```

```suggestion
category: github-action
```

Looks like GHA have their own category: https://heimdall.hashicorp.services/site/assets?category=github-action